docs/getting-started.md

```diff
@@ -0,0 +1,134 @@
+---
+sidebar_position: 1.1
+---
+
+# Getting Started
+
+The `@lukso/lsp6-signer.js` package is used to sign a transaction.
+
+The prefix `\x19LSP6 ExecuteRelayCall:\n` is added to prevent users from signing a transaction while thinking it's a message.
```

Comment:
I think we can go into more detial here. The ExcuteRelayCall prefix is used instead of the standard ethereum transaction to sign messages so that an executeRelayCall transaction cannot be inadvertently signed when signing an ethereum signed message